Add custom data support (#10)
bhftbootcamp/LightweightCharts.jl: README.md

![chart](docs/src/assets/chart.png)

> Custom data types can also be visualized (e.g. time series from [TimeArrays](https://github.com/bhftbootcamp/TimeArrays.jl)). The following example shows how smoothed prices can be visualized:

Reviewer: ```suggestion
The package also support custom data types. In the following example we used [TimeArrays](https://github.com/bhftbootcamp/TimeArrays.jl) to smooth prices:
```
